qemu/tests/cfg/multi_nics_hotplug.cfg

```diff
@@ -24,15 +24,23 @@
             extra_params += "-net none"
         - nic_e1000:
             no ppc64 ppc64le
+            no RHEL.8
             pci_model = e1000
             match_string = "Gigabit Ethernet Controller"
             nics = ""
             extra_params += "-net none"
+        - nic_e1000e:
+            no ppc64 ppc64le
+            pci_model = e1000e
+            nics = ""
+            extra_params += "-net none"
```

Comment:
Not investigate, but I guess here need `match_string`

Reply:
it's not used now.
it should be deprecated code after some code changes previously.